src/pages/en/core-concepts/routing.md

```diff
@@ -19,65 +19,99 @@ src/pages/about/me.astro     -> mysite.com/about/me
 src/pages/posts/1.md         -> mysite.com/posts/1
 ```
 
+> 💡 There is no separate "routing config" to maintain in an Astro project. Static pages are created by placing files in the `/src/pages/` directory.
+
 ## Dynamic routes
 
-Sometimes, you need to generate many URLs from a single page component. Astro uses file-based routing to support **dynamic route parameters** in the filename, so that one page can match many dynamic routes based on some pattern.
+A single Astro Page component can also specify dynamic route parameters in its filename to generate multiple routes that match a given criteria. You can create several related pages at once, such as author pages, or a page for each blog tag.
+
+> 💡 Even dynamically-created pages and routes are generated at build time.
+
+Astro pages that create dynamic routes must: 
+
+1. use `[bracket]` notation to identify the dynamic parameters
+
+2. export a `getStaticPaths()` function to specify exactly which paths will be pre-rendered by Astro.
+
+```astro
+// src/pages/dogs/[dog].astro
+---
+export function getStaticPaths() {
+  const dogs = ["clifford", "rover", "spot"]; //array of routes to be generated
+
+  return dogs.map((dog) => ({
+    params: {
+      dog,
+    },
+  }));
+}
+---
+
+# routes generated:
+mysite.com/dogs/clifford
+mysite.com/dogs/rover
+mysite.com/dogs/spot
+
+```
 
-An important thing to keep in mind: Astro is a static site builder. There is no Astro server to run in production, which means that every page must be built ahead of time. Pages that use dynamic routes must export a `getStaticPaths()` function which will tell Astro exactly what pages to generate. Learn more by viewing the complete [API Reference](/en/reference/api-reference#getstaticpaths).
+📚 Learn more about [`getStaticPaths()`](/en/reference/api-reference#getstaticpaths).
 
 ### Named parameters
 
-Dynamic parameters are encoded into the filename using `[bracket]` notation:
+Routes can be generated from multiple named parameters, at any level of the filepath:
```

Comment:
Reading this, I wasn't sure if this was a new concept, different from Dynamic pages. Maybe start the section with something to tie the two together, like:

"The easiest way to create a dynamic page is to use a dynamic page parameter. [EXPLAIN WHAT THEY ARE AND WHAT THEY DO]"

Reply:
I think I can solve this by taking that heading out and massaging the earlier description. I was going off of the headings from the earlier page, but you're right: it's only causing trouble. It shouldn't be a separate "thing."